---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Physics, Moscow Institute of Physics and Technology, 1993
* M.S. in Physics, Moscow Institute of Physics and Technology, 1995
* Ph.D in Biophysics, Institute of theoretical and experimental biophysics, 2000

Work experience
======
**08.2022-now** Senior Computational Biologist in Biological Systems Group in the 
Okinawa Institute of Science and Technology.\
Analysis of multi-omics data from various environmental and host-related microbial 
communities. Development of new approaches for the mechanistic understanding of the 
principles guiding microbial community evolution.

**11.2015-now** Senior Researcher in the Laboratory of Molecular and Ion Physics in the 
Moscow Institute of Physics and Technology.\
Development of computational infrastructure for collection, storage, retrieval 
and analysis of the brain tumor tissue fragments molecular profiles obtained with 
various ambient mass-spectrometry methods. Development new machine learning and AI
approaches for the tumor boundary detection and estimation of the sample tumor cell 
percentage from the ambient mass-spectrometry spectra.

**07.2019-07.2022** Researcher, School of Biochemistry, University of Liverpool.

**07.2014-11.2019** Head of the laboratory of Mechanisms of genome functioning in the 
Institute of the cell biophysics RAS.

**08.2013-06.2014** Lead researcher in the laboratory of Mechanisms of genome 
functioning  in the Institute of the cell biophysics RAS.

**04.2011-08.2013** Senior researcher in the laboratory of Mechanisms of genome 
functioning  in the Institute of the cell biophysics RAS.

**07.2009-03.2011**  Senior software developer in BioMedCAD project, School
of Informatics, The University of Edinburgh. Development of PRTR
annotation workflow and text-mining backend.

**05.2008-06.2009** Leading Informatics Researcher in ROBuST SABR project,
School of Informatics, The University of Edinburgh.\
Development of experimental data archive and datamart. Development of modeling
pipeline. 

**09.2005-04.2008** Research fellow in Scottish Centre for Systems Biology
and Edinburgh Centre for Bioinformatics School of Informatics The
University of Edinburg.\
Development of Edinburgh Pathway editor and related plug-ins, development 
of Edinburgh Chemical Ontology, development of biological network static properties
analysis algorithm, analysis of human metabolic network, analysis of interferon
signaling pathway. Coordination of SBGN activities in ECB with EPE development,
development of Edinburgh Pathway notation. Development of parameter space
exploration and fitting facility for SBSI. Students supervision.

**08.2005-03.2011** Researcher in the laboratory of Mechanisms of
genome functioning Institute of the cell biophysics RAS\
Analysis of promoter-protein recognition on the basis of E.coli sigma-70
promoter database. Development of algorithm for fast electrostatic
properties of long DNA fragments evaluation. Analysis of electrostatic
profiles of promoter regions of chromosome of *E. coli*.

**04.2002-08.2003**  EMP Project Lead Software Developer\
Development of web service and database for biochemical information for
 human metabolism reconstruction project. 

**09.1999-09.2000** EMP Project Software developer\
Development module for statistical analysis of EMP database.

**05.1998-08.2005** Junior researcher in the laboratory of Mechanisms of
genome functioning Institute of the cell biophysics RAS\
Analysis of promoter-protein recognition on the basis of E.coli sigma-70
promoter database. Development of algorithm for the fast evaluation of electrostatic
properties of long DNA fragments. Analysis of electrostatic
profiles of promoter regions of chromosome of *E. coli*.

**02.1997-02.1998** Visiting student in the Plant Genetics Department,
Weizmann Institute of Science\
Analysis of ligand-protein contacts in the 3-D complexes (LPC software
in any PDB mirror), analysis of influence of type of substitution on the
structure of mutant protein modeling of the 3-D structure of the protein
on the basis of sequence homology.

**09.1995- 02.1997** Post-graduate student in the laboratory of The cell and
tissue grow of the Institute of theoretical and experimental biophysics
RAS.\
Research: investigation of the rhythm of spontaneously beating
cardiomyocytes in vitro.\
Methods:  on-line and off-line signal analysis, signal analysis on the
basis of nonlinear dynamical and statistical methods.
  
Skills
======
* **KNIME** data processing, machine learning, cheminformatics, DB access, node development. Since 2020.
* **Scala**: data management, graph database access. Since 2013.
* **Python**: data management, graph analysis, nonlinear signal analysis, sensitivity analysis, statistical analysis. Since 2012.
* **R/Bioconductor**: data management, SQL-db connectivity, basic sequence analysis, Shiny applications, sensitivity analysis, statistical analysis, package development. Since 2008.
* **Groovy**: Graph analysis, chemoinformatics,  Windows COM, general scripting. Since 2007 
* **Java**: Web-services design on Servlet/JSP, Struts, non-visual server programming; since 2000. Eclipse plug-in and RCP application development; since 2005
* **ANTLR**: Domain-specific language development; since 2008
* **C/C++**:  Normal usage, numerical algorithms, application development using GNU make. Development of code for parallel environments: BlueGene, MPI. Nonlinear analysis, ODE solving, Protein 3D structure analysis etc.; since 1992
* **Fortran**: Usage and modification of legacy software, application development using f77. MPI programming. Protein 3D structure analysis; since 1997
* **Perl**: CGI scripting since 1997
* **Matlab**:  data modelling and analysis; since 1995
* **Octave**:  data modelling and analysis; since 1995
* **Misc** Pascal, Basic
* **Database programming**: ANSI SQL(advanced),PostgreSQL(advanced, custom datatype development), Oracle PL/SQL(advanced), tSQLMS SQL Server(basic), MonetDB (basic)
* **Database administration**: PostgreSQL (base),Oracle (base), MS SQL Server(base), MonetDB (basic)
* **Web design**: HTML, CGI scripts, JavaScript
* **OS**: advanced user Windows, Irix6.1, Linux, MacOSX

Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
Teaching
======
  <ul>{% for post in site.teaching %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
