---
title: "Modelling of bacterial metabolism and metabolic interspecies interactions in reach media"
collection: talks
type: "Plenary talk"
permalink: /talks/2024-08-07-BGRS-talk
venue: "Bioinformatics of Genome Regulation and Structure/Systems Biology, BGRS/SB-2024"
date: 2024-08-07
location: "Novosibirsk, Russian Federation"
---

Since 1999 when the first GEnome-scale Metabolic model (GEM) was published 
[1](https://doi.org/10.1074/jbc.274.25.17410), GEMs have provided an invaluable tool for 
analysis of the physiological capabilities of living organisms. Initially, it was 
single-cell organisms mainly bacteria, but more complex were added later.
Development of 
the human metabolic reconstruction ([2](http://www.nature.com/articles/nbt.2488),
[3](https://onlinelibrary.wiley.com/doi/10.1038/msb4100177)) 
and reorganisation it with tissue-specific submodels into an integrated whole-body model 
([4](https://onlinelibrary.wiley.com/doi/10.15252/msb.20198982)) 
open the avenue for a better understanding of pathological processes such as cancer 
development. Apart from human GEM, several large collections of models were created by 
manual curation ([5](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz1054/5614178)), 
semi-automated reconstruction ([6](https://www.nature.com/articles/s41587-022-01628-0)), 
or fully-automated reconstruction (see in [7](http://www.nature.com/articles/s41587-020-0446-y)) techniques. 

What is common between all such reconstructions, as was noted by Van Pelt-KleinJan with 
coauthors ([7](https://www.mdpi.com/2218-1989/11/5/257)), 
is that all of them are based on physiological data obtained from limited media growth experiments. This is 
reasonable from the methodological point of view, as a single carbon source experiment allows scientists to 
compare experimental growth rate with essentiality data from the model. However, that type of 
model identification imposes some limitations to interpreting the model results in 
a nutrient-rich set-up. The rich environment is very common in biotechnological 
applications, for example in the food industry and when bacteria participate in the 
community and interact with its members via the production and consumption of nutrients. The 
interaction between pathways utilizing different substrates makes solution space more 
complex and increases the number of non-unique solutions to the optimization FBA problem. In 
this talk ([video](https://disk.icgbio.ru/s/3n65Z8WsnGBmgN9)), we propose to solve this problem with 
the new approach ([8](https://www.mdpi.com/2218-273X/13/3/500)), 
which is based on sampling flux space and global sensitivity analysis (GSA), for analysis 
of the GEM FBA solution space in the nutrient-rich environment.

  


